Assets/Placenote/EditorCoroutines/Scripts/EditorCoroutines.cs

```diff
@@ -0,0 +1,392 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System;
+using System.Reflection;
+
```

Comment:
what is this script doing ? I don't see any mesh loading code?

Reply:
These scripts let you run Coroutines (Asynchronous threads) used for downloading metadata etc. without "running" the app i.e. just in the editor